backend/src/test/java/com/woowacourse/levellog/presentation/LevellogControllerTest.java

```diff
@@ -78,6 +79,31 @@ void save_nameNullOrEmpty_Exception() throws Exception {
             perform.andDo(document("levellog/save/exception-contents"));
         }
 
+        @Test
+        @DisplayName("팀의 인터뷰가 시작된 이후에 요청하는 경우 예외를 던진다.")
```

Comment:
```suggestion
        @DisplayName("팀의 인터뷰가 시작된 이후에 레벨로그를 저장하는 경우 예외를 던진다.")
```
와 같이 조금 더 구체적이면 좋을 것 같아요!